### ISSUE TITLE
Update boost package name

### ISSUE THREAD

**Issue by nemolc** [filecoin-project/boost#1943]
### Checklist

- [X] This is **not** a new feature or an enhancement to the Filecoin protocol. If it is,  please open an [FIP issue](https://github.com/filecoin-project/FIPs/blob/master/FIPS/fip-0001.md).
- [X] This is **not** brainstorming ideas. If you have an idea you'd like to discuss, please open a new discussion on [the Boost forum](https://github.com/filecoin-project/boost/discussions/categories/ideas) and select the category as `Ideas`.
- [X] I **have** a specific, actionable, and well motivated feature request to propose.

### Boost component

- [X] boost daemon - storage providers
- [X] boost client
- [X] boost UI
- [X] boost data-transfer
- [X] boost index-provider
- [ ] Other

### What is the motivation behind this feature request? Is your feature request related to a problem? Please describe.

I am writting some automation tools that are dependent on "github.com/filecoin-project/boost".
I want to use the latest version.but it is diffcult to make use of version 2 of package.

### Describe the solution you'd like

 will package name be changed from "github.com/filecoin-project/boost" to "github.com/filecoin-project/boost/v2"?

**Comment by LexLuthr** [comment]
No. Boost is no longer being actively developed and would be deprecated in coming year. 
If you can provide me details about your requirement, I might be able to help with a different approach.
